projects/igniteui-angular/src/lib/query-builder/query-builder-tree.component.html

```diff
@@ -356,7 +356,7 @@
                                             #searchValueInput
                                             igxInput
                                             [disabled]="isSearchValueInputDisabled()"
-                                            [placeholder]="this.selectedCondition === 'in' || this.selectedCondition === 'notIn' ? 
+                                            [placeholder]="this.selectedCondition === 'inQuery' || this.selectedCondition === 'notInQuery' ? 
```

Comment:
This seems to be triggered by nested query, wouldn't it better check for the `isNestedQuery`?